File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    do
    {
        size = get_int("Choose size of the pyramid (1-8): ");
    } while (size<1 || size>8);

    //Make a line
    for (int i=0; i<size; i++)
    {
        printf("#");
    }
    //Make next row
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

def get_height():
    while True:
        height = get_int("What's pyramid height? ")
        if height > 0 and height <= 8:
            return height

def printpyramid(n):
    size = n
    row = size
    for i in range(size):
        print(" ", end="")
    print("#", end="")
    print(" ", end="")
    for i in range(n-1):
        print("#")

def main():
    h = get_height()
    printpyramid(h)

main()